engine/src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -4,13 +4,14 @@ import java.io.{PrintWriter, StringWriter}
 
 import cromwell.engine.CromwellFatalException
 import cromwell.logging.WorkflowLogger
+import org.apache.commons.lang3.exception.ExceptionUtils
 
 import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
 case class AggregatedException(exceptions: Seq[Throwable], prefixError: String = "") extends Exception {
   override def getMessage: String = {
-    prefixError + exceptions.map(_.getMessage).mkString("\n")
+    prefixError + exceptions.map(e => s"${e.getMessage}\n${ExceptionUtils.getStackTrace(e)}").mkString("\n")
```

Comment:
Moving the stack traces into the message… For some reason this horrifies and impresses me at the same time.